models/VolunteerLogs.ts

```diff
@@ -0,0 +1,15 @@
+import mongoose from 'mongoose';
+
+// VolunteerLogSchema describes what our documents should look like in our VolunteerLogs collections
+const VolunteerLogSchema = new mongoose.Schema({
+  school: { type: String, required: true },
+  teacher: { type: String, required: true },
```

Comment:
We're currently unsure if `school` and `teacher` should be a required field, but hopefully things will be clearer after our meeting with stakeholder today.

Reply:
Verdict: changing both `school` and `teacher` to optional because we have programs like BFNK that don't require them.